modules/chat.py

```diff
@@ -17,6 +17,14 @@
 from modules.text_generation import (encode, generate_reply,
                                      get_max_prompt_length)
 
+#put stopping_strings up here so its easier to add more, 3rd and 4th are for vicuna
+def stopping_strings(name1, name2):
+    return [
+        f"\n{name1}:",
+        f"\n{name2}:",
+        f"\n### Human:",
+        f"\n### Assistant:"
```

Comment:
Isn't this culture (English) specific? Should we maybe provide a file containing the _stopping_ words?!